ee/clickhouse/process_event.py

```diff
@@ -181,19 +180,21 @@ def log_event(
     now: datetime.datetime,
     sent_at: Optional[datetime.datetime],
     event_uuid: UUIDT,
+    *,
+    topics: Sequence[str],
```

Comment:
I think keyword parameters are a win most of the time. I'm a fan of Swift' approach BTW: https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID526 Really nice stuff, keyed by default so omitting is the optional way, _and_ you can define a name for internal use than for the external use, just in case you want to make things to sound more natural.